torch/_dynamo/backends/common.py

```diff
@@ -70,3 +73,37 @@ def mem_efficient_fusion_kwargs(use_decomps):
         kwargs["decompositions"] = default_decompositions
 
     return kwargs
+
+
+def fake_tensor_unsupported(fn):
+    """
+    Decorator for backends that need real inputs.  We swap out fake
+    tensors for zero tensors.
+    """
+
+    def defake(x):
+        if not isinstance(x, FakeTensor):
+            return x
+        y = torch.empty_strided(
+            x.size(),
+            x.stride(),
+            dtype=x.dtype,
+            device=x.device,
+            requires_grad=x.requires_grad,
+        )
+        y.zero_()
```

Comment:
layout?

Reply:
zero_ doesn't change the layout set above.

Reply:
I don't see you passing layout to empty_strided though?